picamera2/devices/imx500/imx500.py

```diff
@@ -568,14 +568,14 @@ def __get_input_tensor_info(self, tensor_info) -> tuple[str, int, int, int]:
 
     @staticmethod
     def get_kpi_info(metadata: dict) -> Optional[tuple[float, float]]:
-        """Return the KPI parameters in the form (dnn_runtime, dsp_runtime)."""
+        """
+            Return the KPI parameters in the form (dnn_runtime, dsp_runtime).
+            the numbers are in milliseconds
+        """
         kpi_info = metadata.get('CnnKpiInfo')
         if kpi_info is None:
             return None
-        if type(kpi_info) not in [bytes, bytearray]:
-            kpi_info = bytes(kpi_info)
-
-        dnn_runtime, dsp_runtime = struct.unpack('II', kpi_info)
```

Comment:
yes 
is works back than 
but someting is change 
and now `kpi_info = metadata.get('CnnKpiInfo')` 
`kpi_info` is tuple of int

Reply:
Oh, I think this may have changed with some work @davidplowman recently did on controls.  Change looks fine to me, but would you be able to update the commit message to something like:

```
imx500: Fix IMX500.get_kpi_info()

Pull the KPI values directly from the tuple instead of decoding a byte array.

Signed-off-by: Your Name <your@gmail.com>
```